src/Control/Monad/Freer/Reader.hs

```diff
@@ -73,6 +76,17 @@ asks f = f <$> ask
 runReader :: Eff (Reader e ': effs) a -> e -> Eff effs a
 runReader m e = handleRelay pure (\Reader k -> k e) m
 
+-- | `runReader` with the arguments flipped. May be more convenient for
+-- chaining `run*` commands together.
+--
+-- @
+-- eff :: (Member (Reader Env) effs, Member (Writer String) effs) => Eff effs ()
+--
+-- foo :: ((), String)
+-- foo = run . runReader env $ runWriter eff
+runReader' :: e -> Eff (Reader e ': effs) a -> Eff effs a
```

Comment:
I now think using `flip` in user code is the better idea. I'll change this to add an example showing that.